cpp/include/raft/neighbors/ivf_flat.cuh

```diff
@@ -469,6 +469,31 @@ void search(raft::device_resources const& handle,
                 nullptr);
 }
 
+/**
+ * @brief Reconstruction of the index
+ *
+ * @tparam T data element type
+ * @tparam IdxT type of the indices in the source dataset
+ *
+ * @param[in] handle
+ * @param[in] idx
+ * @param[in] vector_ids a mdspan array containing the ids of vectors to be reconstructed [n_rows]
+ * @param[out] vector_out a mdspan array to hold the vectors to be reconstructed [n_rows,
+ * index.dim()]
+ */
+template <typename T, typename IdxT>
```

Comment:
Can `value_t` and `idx_t` be used for the template names so that all the new public mdspan API functions use the same type name.

Reply:
I'd suggest to **not** do this until we change the corresponding [.clang-tidy](https://github.com/rapidsai/raft/blob/76c828dd4da4dc922626ba2a440a46dea6ab03b9/cpp/.clang-tidy#L211). Note also that we use camel case in plenty of public/mdspan api, including the ivf-pq that mirrors the IVF-Flat interface.